README.md

```diff
@@ -77,4 +81,11 @@ curl --request POST \
   --header 'content-type: application/json' \
   --data '{"client_id":"0FhpaZeIjhSG1lwNR3RWPI20VgLgU5rk",
             "client_secret":<client_secret>,
-            "audience":"https://findme-dev.eu.auth0.com/api/v2/","grant_type":"client_credentials"}'
\ No newline at end of file
+            "audience":"https://findme-dev.eu.auth0.com/api/v2/","grant_type":"client_credentials"}'
+```
+
+
+### Documentation
+For the detailed api documentation start the application as described above and visit the following endpoints:
+  - `{your_application_url}/users/swagger`
```

Comment:
minor: can we combine docs in common path? e.g {api_endpoint_url}/docs @v4lentin1879